public/apis/model.ts

```diff
@@ -16,6 +16,7 @@ import { InnerHttpProvider } from './inner_http_provider';
 export interface ModelSearchItem {
   id: string;
   name: string;
+  description: string;
```

Comment:
Is that description represent the version notes? Is that field name confirmed with backend side?

Reply:
Good question, the description is from the current API response, the new version details API may not have it. I'll leave a TODO item and fix it when integrating the API changes.